services/frontend/src/components/Users/UserSearchList.jsx

```diff
@@ -45,101 +46,174 @@ export const UserSearchList = () => {
     setPopupTimeout(null)
   }
 
+  const columnsWithIamGroups = [
+    {
+      key: 'NAME',
+      title: 'Name',
+      getRowVal: user => {
+        const nameparts = user.name.split(' ')
+        return nameparts[nameparts.length - 1]
+      },
+      getRowContent: user => user.name,
```

Comment:
Could we instead do something like this to simplify the code somewhat:

  const columsWithIamGroups = [
    ...columnsWithoutIamGroups.slice(0, 4),
    {
      key: 'IAMGROUPS',
      title: 'IAM groups',
      sortable: false,
      filterType: 'multi',
      getRowContent: user => (
        <Label.Group>
          {user.iamGroups.toSorted().map(iam => (
            <Label content={iam} key={iam} />
          ))}
        </Label.Group>
      ),
      getRowVal: user => user.iamGroups,
    },
    ...columnsWithoutIamGroups.slice(4),
  ]

Reply:
Yup changed